src/colorscheme.rs

```diff
@@ -118,27 +119,43 @@ fn convert_color(raw: i64) -> Color {
 	}
 }
 
-fn parse_colorscheme(config_folder: &Path, colorscheme: &Colorschemes) -> ColorschemeRaw {
+fn get_colorscheme_config(config_folder: &Path, colorscheme: &Colorschemes) -> String {
 	match colorscheme {
-		Colorschemes::Custom(name) => serde_json::from_str(
-			&fs::read_to_string(config_folder.join(name).with_extension("json")).unwrap(),
```

Comment:
I think a cleaner way to implement this would be to just change these unwraps. I don't think we need to change any of these functions.

Reply:
I've updated the PR and the code should be much cleaner now.